src/main/java/com/example/usermanageapirest/domain/services/input/UserCreateInput.java

```diff
@@ -0,0 +1,23 @@
+package com.example.usermanageapirest.domain.services.input;
+
+import lombok.Data;
+import lombok.experimental.Accessors;
+
+@Accessors(chain = true)
+@Data
+public class UserCreateInput {
+
+	private String firstname;
+
+	private String lastname;
+
+	private int age;
+
+	private String language;
+
+	private int rank;
+
+	private String country;
+
+	private String active;
```

Comment:
Lo mismo respecto a los espacio, mira el cambio sugerido:

```suggestion
	private String firstname;
	private String lastname;
	private int age;
	private String language;
	private int rank;
	private String country;
	private String active;
```